examples/README.md

```diff
@@ -0,0 +1,34 @@
+# How to run
+
+1. Realtime
+
+- Go to realtime dir `cd realtime`
+- Rename **.env.example** to **.env**, update `ABLY_KEY` with your own key 
+- Realtime channel pub-sub
+
+    ```go run presence.go constants.go```
```

Comment:
Inline code snippets are enclosed in single quotes,

    `like this`
    
Triple backticks are for code blocks, and go in their own line.

    ```
    like this 
    ```

([Source](https://guides.github.com/features/mastering-markdown))